src/app/install/sql/mysql.sql

```diff
@@ -74,7 +74,7 @@ ENGINE=InnoDB;
 create table usvn_users
 (
    users_id                       int                            not null auto_increment,
-   users_login                    varchar(255)                   not null,
+   users_login                    varchar(127)                   not null,
```

Comment:
What is the benefits of reducing the size?

Reply:
In the recent mysql version, the default character set is utf8mb4. If you keep UNIQUE key length as 255, the key length error will appear. It's something like "Specified key was too long; max key length is 767 bytes". I think the length of 127 is enougth, it's the most simple way to solve the problem. Your won't have to change the config of mysql.

Reply:
Thanks